src/page-modules/contact/components/input/input.module.css

```
@@ -36,3 +35,11 @@
   box-shadow: inset 0 0 0 var(--border-width-medium)
     var(--interactive-interactive_2-outline-background);
 }
+
+.error {
+  color: red;
+}
+
+.input_error {
+  border-color: red;
+}
```

To align more with other css files I suggest to use: 
```suggestion
.input_label__error {
  color: red;
}

.input__error {
  border-color: red;
}
```